src/common/Utils.ts

```diff
@@ -123,3 +123,12 @@ export function isCustomTelemetryEnabled():boolean {
         .get(CUSTOM_TELEMETRY_FOR_POWER_PAGES_SETTING_NAME);
     return isCustomTelemetryEnabled as boolean;
 }
+
+export function getUserAgent(): string {
+    const userAgent = "{product} / {product-version} {comment}";
+
+    return userAgent
+        .replace("{product}", EXTENSION_NAME)
+        .replace("{product-version}", getExtensionVersion())
+        .replace("{comment}", getExtensionType()+'; '+getExtensionVersion());
```

Comment:
See how browsers do this. The comment, if specified is usually in parentheses. This makes it so spaces can be inside the comment.
i.e. `"myproduct/0.1.0-dev (extVer:5.0; otherThing:value; flagValue)"`

Reply:
put it in parenthesis